index.d.ts

```diff
@@ -0,0 +1 @@
+declare function T(strings: TemplateStringsArray, ...args: any[]): string;
```

Comment:
Would you mind to explain how it works? Maybe attach links to the docs or something.

Reply:
It's just valid definition of tag function. You can see examples here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates

Definitions of TemplateStringsArray you can find here: https://github.com/microsoft/TypeScript/blob/9ff24b6fc87dd4a376a434d1019d356dfe743c53/lib/lib.es5.d.ts#L604

And examples of using it are here: https://github.com/microsoft/TypeScript/blob/c6c2c4c8d5aa0947de16f484b8c16fb0eab1c48f/tests/baselines/reference/taggedTemplatesInDifferentScopes.js#L2-L4

Reply:
Great, thanks!